data/_ui/tooltips.txt

```diff
@@ -1531,3 +1531,15 @@ tip "Show parenthesis"
 
 tip "Notify on destination"
 	`If you enter a system with a planet you must land on for a mission, plays a notification sound.`
+
+tip `Override "universal ramscoop"`
+	`Replace the "universal ramscoop" gamerule provided by the game and plugins with a custom value.`
+
+tip `"universal ramscoop" value`
```

Comment:
We should not have 2 separate settings to effectively change what looks to be 1 setting with options; "**default**", **"always on"**, and **"always off"**

Reply:
I also considered that, but I thought having a common two-settings pattern for all gamerules would be better.

Reply:
I wonder if we should even make the "universal ramscoop" available through settings. Casual players just don't know that a ramscoop is, and I feel that it's somehow better if we just make an informed decision on what we want it to be, and then only allow overriding by a plugin, and not allow any override by settings.